pcsx2/gui/MemoryCardFile.cpp

```diff
@@ -206,6 +314,18 @@ void FileMemoryCard::Open()
 		NTFS_CompressFile( str, g_Conf->McdCompressNTFS );
 #endif
 
+		if (str.EndsWith(".bin"))
+		{
+			wxString newname = str + "x";
+			if(!ConvertNoECCtoRAW(str, newname))
+			{
```

Comment:
put the bracket on the same line as the if to match the rest of the file, also space between if and (